src/coreclr/build-runtime.sh

```diff
@@ -171,6 +171,26 @@ if [[ -n "$__RequestedBuildComponents" ]]; then
     __CMakeTarget=" $__RequestedBuildComponents "
     __CMakeTarget="${__CMakeTarget// paltests / paltests_install }"
 fi
+
+if [[ "$__CMakeTarget" == *"wasmjit"* ]]; then
+  __ExtraCmakeArgs="$__ExtraCmakeArgs -DCLR_CMAKE_BUILD_LLVM_JIT=1"
```

Comment:
Do we need `__ExtraCmakeArgs` in addition to `__CMakeArgs`?

Reply:
missed this one, but is in now.